test/suites/container_devices_proxy.sh

```diff
@@ -692,3 +693,70 @@ container_devices_proxy_tcp_udp() {
   # Cleanup
   lxc delete -f proxyTester
 }
+
+container_devices_proxy_with_overlapping_forward_net() {
+  echo "====> Testing proxy creation with overlapping network forward"
+  ensure_import_testimage
+  ensure_has_localhost_remote "${LXD_ADDR}"
+
+  netName="testnet"
+
+  lxc network create "${netName}" \
+        ipv4.address=192.0.2.1/24 \
+        ipv6.address=fd42:4242:4242:1010::1/64
+
+  overlappingAddr="192.0.2.2"
+  proxyTesterStaticIP="192.0.2.3"
+  HOST_TCP_PORT=$(local_tcp_port)
+
+  # First, launch container with a static IP
+  lxc launch testimage proxyTester
+  lxc config device add proxyTester eth0 nic \
+    nictype=bridged \
+    name=eth0 \
+    parent=${netName} \
+    ipv4.address=${proxyTesterStaticIP}
+
+  # Then, we set up network forward
+  firewallDriver=$(lxc info | awk -F ":" '/firewall:/{gsub(/ /, "", $0); print $2}')
```

Comment:
We dont need this bit as we are not testing the network forward functionality, only whether its listen address is overlapping with proxy devices.